stdlib/nodejs/file.rb

```diff
@@ -59,102 +59,111 @@ class File < IO
     ALT_SEPARATOR = `__path__.sep`
   end
 
-  def self.read(path)
-    `return executeIOAction(function(){return __fs__.readFileSync(#{path}).toString()})`
-  end
+  class << self
```

Comment:
I'm using `class << self`` to remove `def.self` redundancy but I can revert this change if neededed

Reply:
Yes please, keep the explicit `self.`, it's easier to recognize singleton methods in long method lists this way, also keeps the diff cleaner 😉

Reply:
Ok